pumpkin-lib/src/propagators/arithmetic/maximum.rs

```diff
@@ -67,23 +69,120 @@ impl<ElementVar: IntegerVariable, Rhs: IntegerVariable> Propagator
         &self,
         mut context: PropagationContextMut,
     ) -> PropagationStatusCP {
+        // This is the constraint that is being propagated:
+        // max(a0, a1, ..., an-1) = rhs
+
+        // Propagate bounds on the variables based on the rhs:
```

Comment:
Does it make sense to only describe the rules where they are implemented? Already this comment is out of date (rule 5 is not in here).

Reply:
I made this change to the new version.